test/powershell/Modules/Microsoft.PowerShell.Utility/string.tests.ps1

```diff
@@ -14,7 +14,8 @@
             $pathWithoutWildcard = $TestDrive
             $pathWithWildcard = Join-Path $TestDrive '*'
 
-            $tempFile = New-TemporaryFile
+            # Here Get-ChildItem add 'PSDrive' property
```

Comment:
I had trouble understanding this comment. Can you correct `add` to `adds`?

Reply:
Fixed.